src/client.js

```diff
@@ -0,0 +1,30 @@
+import Dashcore from '@dashevo/dashcore-lib'
+
+export default class Client {
+  constructor (socket) {
+    this.socket = socket
+  }
+
+  #modifiedTransaction (tx) {
+    return {
+      txId: tx.hash,
+      version: tx.version,
+      type: tx.type,
+      size: tx.toBuffer().length,
+      locktime: tx.nLockTime,
+      from: tx.inputs.map(input => input.script?.toAddress(Dashcore.Networks.testnet).toString()),
```

Comment:
We should define network at the start up / init phase. Create an `NETWORK` environment, then you can use proper address for different networks